OSDOCS-18782 adding ESO docs to 4.19 for GA (#109778)
openshift/openshift-docs: modules/external-secrets-operand-install-cli.adoc

:_mod-docs-content-type: PROCEDURE
[id="external-secrets-operand-install-cli_{context}"]

> = Installing the External Secrets operand by using the CLI

Reviewer: ```suggestion
= Installing the External Secrets Operand by using the CLI
```

Author: Done

[role="_abstract"]

> Install the External Secrets operand on {product-title} by using the CLI to create the necessary configuration object. By completing this task, you ensure that the {external-secrets-operator-short} is properly configured to manage secrets from external APIs on your cluster.

Reviewer: ```suggestion
Ensure that the {external-secrets-operator-short} is properly configured to manage secrets from external APIs on your cluster....etc
```
job first, how second

Author: Done

.Prerequisites

* You have access to the cluster with `cluster-admin` privileges.

.Procedure

. Create an `externalsecretsconfig.openshift.operator.io` object by defining a YAML file with the following content:
+

> .Example `externalsecretsconfig.yaml` file.

Author: Done

$ oc get pods -n external-secrets
----
+

> The following example shows that the `external-secrets` pods are running.

Reviewer: example output is correct; revert
